tokio/Cargo.toml

```diff
@@ -84,6 +84,7 @@ signal = [
 ]
 stream = ["futures-core"]
 sync = ["fnv"]
+syscall = []
```

Comment:
I think we should include this as part of `test-util` instead of creating a new feature flag.

Reply:
Hmm I think this would mean anyone using `test-util` will need to provide a `Syscall` impl. Is the idea that we'll have a default `Syscall` impl for the case where users want to test their code but not necessarily with simulated networking/DNS?

Reply:
How would a default `Syscall` impl work in that case? If the user would still have to do something to make it work, then including this in `test-util` is a breaking change...

Reply:
There should be a default `Syscall` implementation which maintains behavior as is. Features must be additive anyway, so a `syscall` feature flag cannot really change behavior.